### PR TITLE
REST fixes for tags and services

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/configure/test_tag.py -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 87e916563f599b3c610d4ab4480ca848e9712055:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f4e4eedf25edfd6aaf56c860f55bd97c2693833d:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0d9e38cf1e811238a28b5c9416f22021124788a8:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 415f971c521438812b14d8a91045e4c5ab4de8ff:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 69b702523b243b067d9a385b85a3a5b31e5a7d9d:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3f8a764e8fa2e713694b598c6e80c29bc50ee3ca:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e54cf451073ae379bbc94bb8ec8113b796e71e82:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b6cc5b83568691d7129c2ac7a6a1c1c6c5f57892:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5e0ad7981d89d3d0953a55d9b40152db4276a1d0:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5973658a2bb6d4fd3dd02f3778e6baada09f84d4:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/configure/configuration/__init__.py [hunk 1]
```diff
@@ -1782,6 +1783,8 @@ def _form_mapping(self, create=None, **kwargs):
 
     def create(self):
         sel.force_navigate("cfg_settings_region_my_company_tag_new", context={"tag": self})
+        if current_version() < "5.6":
+            sel.click(tag_form.new)
```
Comment (jkrocil) [inline cfme/configure/configuration/__init__.py]: Shouldn't be necessary if you rebase on latest master.

File: cfme/configure/configuration/__init__.py [hunk 2]
```diff
@@ -1791,6 +1794,9 @@ def update(self, updates):
             version.LOWEST: tag_form.add,
             '5.6': tag_form.save
         })
+        if current_version() < "5.6":
+            row = classification_table.find_row_by_cells({'name': self.name})
+            sel.click(row[1])
```
Comment (jkrocil) [inline cfme/configure/configuration/__init__.py]: Same here.

File: cfme/configure/configuration/__init__.py [hunk 3]
```diff
@@ -1799,6 +1805,9 @@ def delete(self, cancel=True):
         if not cancel:
             sel.force_navigate("cfg_settings_region_my_company_tags")
             fill(tag_form, {'category': self.category.display_name})
+            # Workaround for testing purpose
+            from time import sleep
+            sleep(5)
```
Comment (jkrocil) [inline cfme/configure/configuration/__init__.py]: Same here.

File: cfme/rest/__init__.py [hunk 4]
```diff
@@ -11,81 +11,44 @@
 from utils.wait import wait_for
 
 
-def service_catalogs(request, rest_api):
+def service_catalogs(request, rest_api, num=1):
```
Comment (jkrocil) [inline cfme/rest/__init__.py]: Any function defined outside of `cfme/tests` shoud not need `request`.

**Comment by dajoRH** [comment]
Lint report for commit 418ef1e4b91ff1e2dc825941c1750e4bff23f509:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1eab27e029baeb8987fb44bbfce24b7234419993:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request closed without merging**